sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -1478,56 +1453,33 @@ Mono<Response<QueueProperties>> createQueueWithResponse(String queueName, Create
      */
     Mono<Response<RuleProperties>> createRuleWithResponse(String topicName, String subscriptionName, String ruleName,
         CreateRuleOptions ruleOptions, Context context) {
-        if (topicName == null) {
-            return monoError(LOGGER, new NullPointerException("'topicName' cannot be null."));
-        } else if (topicName.isEmpty()) {
-            return monoError(LOGGER, new IllegalArgumentException("'topicName' cannot be empty."));
+        if (CoreUtils.isNullOrEmpty(topicName)) {
+            return monoError(LOGGER, new NullPointerException("'topicName' cannot be null or empty."));
```

Comment:
In the example above, we are throwing an IllegalArugmentException, but here it is a NullPointerException.  Similarly with the ones below... should it be NPE or IllegalArgument?

Reply:
May have to update the javadocs to make sure they match.